hi-ml-histopathology/src/histopathology/models/deepmil.py

```diff
@@ -320,17 +324,23 @@ def update_results_with_data_specific_info(self, batch: dict, results: dict) ->
 class SlidesDeepMILModule(BaseDeepMILModule):
     """Base class for slides based deep multiple-instance learning."""
 
-    def __init__(self, tiles_count: int, **kwargs: Any) -> None:
-        self.tiles_count = tiles_count
-        super().__init__(**kwargs)
-
     @staticmethod
     def get_bag_label(labels: Tensor) -> Tensor:
         # SlidesDataModule attributes a single label to a bag of tiles already no need to do majority voting
         return labels
 
     def update_results_with_data_specific_info(self, batch: dict, results: dict) -> None:
         # WARNING: This is a dummy input until we figure out tiles coordinates retrieval in the next iteration.
-        results.update({ResultsKey.SLIDE_ID: [batch[SlidesDataset.SLIDE_ID_COLUMN] * self.tiles_count],
-                        ResultsKey.TILE_ID: [batch[SlidesDataset.SLIDE_ID_COLUMN] * self.tiles_count],
-                        ResultsKey.IMAGE_PATH: [batch[SlideKey.IMAGE_PATH] * self.tiles_count]})
+        results.update(
+            {
+                ResultsKey.SLIDE_ID: [
+                    [slide_id] * self.tile_count for slide_id in batch[SlideKey.SLIDE_ID]
+                ],
+                ResultsKey.TILE_ID: [
+                    [f"_{tile_id}" for tile_id in range(self.tile_count)] for _ in batch[SlideKey.SLIDE_ID]
```

Comment:
Agree! However, this is just a "dummy" filling of the results... tile_id here is not really an id per se it's more like an index in range(self.tile_count). We don't really have a proper tile_id at this point because of the current way of tiling on the fly. We somehow loose the tiles order due to shuffling and random_offset... I wasn't really sure what to put here, I just tried to fill it in with the expected shapes so that it's stops complaining. We have to revisit that once we resolve coordinates retrieval.